source/manual/taxonomy.html.md

```diff
@@ -73,6 +73,23 @@ tagged to the above mentioned ["Education taxon"][education-taxon]:
 
 [https://www.gov.uk/api/search.json?filter_taxons[]=c58fdadd-7743-46d6-9629-90bb3ccc4ef0](https://www.gov.uk/api/search.json?filter_taxons[]=c58fdadd-7743-46d6-9629-90bb3ccc4ef0)
 
+By default rummager returns a handful of fields in a search result item.
+There are many more fields that a search result can have and you can tell
+rummager which ones to include in the results by passing the field names
+in an `fields` array to the search parameters:
+
+[https://www.gov.uk/api/search.json?filter_taxons[]=c58fdadd-7743-46d6-9629-90bb3ccc4ef0&fields[]=title,description,link,content_store_document_type,format,organisations](https://www.gov.uk/api/search.json?filter_taxons[]=c58fdadd-7743-46d6-9629-90bb3ccc4ef0&fields[]=title,description,link,content_store_document_type,format,organisations)
+
+*If a content item does not have one of the named fields provided,
+it will be left out of the returned item. There is a list of all the fields[here][fields]*
+
+You can filter on multiple different field names if you wish to narrow
+down what you are searching for. You are able to filter by one or many
+field names and multiple values can be given for each field name.
+To find all `speech` content tagged to ["Education taxon"][education-taxon]:
+
+[https://www.gov.uk/api/search.json?filter_taxons[]=c58fdadd-7743-46d6-9629-90bb3ccc4ef0&filter_content_store_document_type[]=speech&fields[]=title,description,link,content_store_document_type,format](https://www.gov.uk/api/search.json?filter_taxons[]=c58fdadd-7743-46d6-9629-90bb3ccc4ef0&filter_content_store_document_type[]=speech&fields[]=title,description,link,content_store_document_type,format)
+
```

Comment:
I think it might be better to try and link to the existing Rummager documentation (e.g. [1]), as it's a general tool, and not specific to the Taxonomy?

1: https://docs.publishing.service.gov.uk/apis/search/search-api.html#returning-specific-document-fields

Reply:
Thats a good point. Will do.